### ISSUE TITLE
example repo showing basic usage

### ISSUE THREAD

**Issue by nelsonic** [dwyl/ampl#30]
we need a basic usage example repo to demo this.

**Comment by des-des** [comment]
ideas?

**Comment by nelsonic** [comment]
I would prefer to just *use* this for publishing a dwyl blog so https://github.com/dwyl/dwyl.github.io would be the example...

**Comment by des-des** [comment]
@nelsonic. Cool. With a build -> push to ghpages step, or something more dynamic?

**Comment by iteles** [comment]
@des-des We're actually not using gh-pages for the dwyl site, confusingly!

**Comment by nelsonic** [comment]
@des-des I don't have anything *against* using GHPages

**Comment by jruts** [comment]
What about a react component that lets you you write md5 on the left and then renders the amped html on the right? Maybe with some extra input fields for all the options.

**Comment by nelsonic** [comment]
if you have *time* please do an awesome demo and this time *invoice* us for your time! (*even if you enjoy doing it*!)

**Comment by des-des** [comment]
@nelsonic yeah im super keen. If we can get a spec together.. If you are keen for a dwyl blog that sounds awesome. We could host the content on gh? if we did I would suggest gh hook (on dwyl/blog repo?) -> download -> parse -> save then serve with static file server? or something else?
right now we have [this](http://ampl-demo.herokuapp.com) and [this](http://ampl-demo.herokuapp.com/dwyl/hapi-auth-jwt2)

**Comment by des-des** [comment]
@nelsonic or easier, keep a blog folder in the `dwyl.github.io` repo then just build that..
